test/lib.bash

```diff
@@ -198,7 +198,6 @@ function run_rolling_upgrade_tests {
   export EVENTING_UPGRADE_TESTS_CONFIGMOUNTPOINT=/.config/wathola
   export GATEWAY_OVERRIDE="kourier"
   export GATEWAY_NAMESPACE_OVERRIDE="${INGRESS_NAMESPACE}"
-  export GO_TEST_VERBOSITY=standard-verbose
```

Comment:
Why this? I'm curious what's changed with that.

Reply:
That's my reasoning https://github.com/openshift-knative/serverless-operator/pull/1517/commits/fc67f579a93ae9dbca4df529de58381efb4a9d53

> Remove verbose logging since it's non debuggable

I was having a hard time following all the MB of logs, but I can revert that commit